webapp/src/components/issue_attribute_selector.tsx

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useState, useEffect} from 'react';
+import ReactSelect, {OnChangeValue} from 'react-select';
+import {Theme} from 'mattermost-redux/types/preferences';
+
+import {getStyleForReactSelect} from 'src/utils/styles';
+import {usePrevious} from 'src/hooks/use_previous';
+
+import Setting from './setting';
+
+type PropTypes = {
+    isMulti: boolean;
+    projectName: string;
+    theme: Theme;
+    label: string;
+    onChange: (value: OnChangeType) => void;
+    loadOptions: () => Promise<Array<SelectionType>>,
+    selection: OnChangeType;
+};
+
+const IssueAttributeSelector = ({isMulti, projectName, theme, label, onChange, loadOptions, selection}: PropTypes) => {
+    const [options, setOptions] = useState<SelectionType[]>([]);
```

Comment:
I can't tell what `SelectionType` is or where it's defined

Reply:
It is defined in the types folder under common types https://github.com/Brightscout/mattermost-plugin-gitlab/blob/1f9fbc3912fb9b8f5f7f679dff259cb44a63efa1/webapp/src/types/common.d.ts#L3